src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,70 @@
+package codeu.chat.util;
+
+import java.util.LinkedList;
+
+public final class Tokenizer {
+  private LinkedList<String> tokens = new LinkedList<String>();
+
+  public Tokenizer(String line) {
+    StringBuilder token = new StringBuilder();
+    boolean inQuotes = false;
+    boolean lookingForEscapable = false;
+
+    for (int i = 0; i < line.length(); i++) {
+      char c = line.charAt(i);
+      if (lookingForEscapable) {
```

Comment:
It looks like you're adding some additional functionality beyond what has been specified, allowing the user to input escaped characters. If you want to do it for fun, that's cool, but it's not needed.